lib/average.js

```diff
@@ -0,0 +1,36 @@
+class Average {
```

Comment:
Nice ES6 class! I might go for a more descriptive name, like `ArrayStats` or something, since `Average` does a lot more than just calculate the average.